ckanext/recombinant/read_excel.py

```diff
@@ -35,6 +36,12 @@ def read_excel(f: Union[str, FlaskFileStorage, FieldStorage],
     for sheetname in wb.sheetnames:
         if sheetname == 'reference':
             return
```

Comment:
this could cause us a problem if someone moved the reference sheet ahead of the data sheets. We could remove it with the change below